gulpfile.js

```diff
@@ -0,0 +1,140 @@
+const exec = require('child_process').exec;
+const fs = require('fs');
+const readline = require('readline');
+
+const cwsupload = require('chrome-webstore-upload');
+const eslint = require('gulp-eslint');
+const {google} = require('googleapis');
+const gulp = require('gulp');
+const jeditor = require('gulp-json-editor');
+const zip = require('gulp-zip');
+
+const CWS_ID = 'aojcceglbipehndciapjedoomockgagl';
+
+/**
+ * Checks the code for style errors.
+ */
+gulp.task('lint', () => {
+  return gulp.src(['src/**/*.js', 'gulpfile.js', '!node_modules/**'])
+      .pipe(eslint({
+        baseConfig: require('./.eslintrc.js')
+      }))
+      .pipe(eslint.format())
+      .pipe(eslint.failAfterError());
+});
+
+/**
+ * Updates the Chrome Web Store manifest to point to the latest version of the
+ * scriipt.
+ */
+gulp.task('latest', () => {
+  // Determine the latest version of the script.
+  exec('clasp versions', (err, out) => {
+    if (err) return console.error(err);
+    const line = out.trim().split('\n').pop();
+    const version = line.split(' ').shift();
```

Comment:
`shift()` is not really necessary here, `[0]` is more readable IMO.

Reply:
Done